### PR TITLE
Button Links

### PR DESCRIPTION
**Pull request by Savith2000**

### CODE REVIEW

File: controllers/surveyController.js [hunk 1]
```diff
@@ -1,48 +1,65 @@
 const db = require("../models");
 
+<<<<<<< HEAD
```
Comment (ka930) [inline controllers/surveyController.js:3]: Why aren't you fixing the merge conflict?

**Review by ka930 — CHANGES_REQUESTED** [review]

File: models/Survey.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const Schema = mongoose.Schema;
 const ALLOWEDTYPES = ["text", "number", "rating"];
 
 const SurveyQuestion = new Schema({
-  question: { type: String, required: true, maxLength: 200, match: [/([A-z0-9\s])+/g, "Special characters are not allowed"] },
```
Comment (ka930) [inline models/Survey.js:8]: Why are you reverting the changes here??

**Review by aidanobryan — COMMENTED** [review]
This would overwrite important changes in main and the app would not work

File: package-lock.json [hunk 3]
```diff
@@ -1,6 +1,10 @@
 {
   "name": "surveyApp",
   "version": "1.0.0",
+<<<<<<< HEAD
```
Comment (ka930) [inline package-lock.json:4]: Why aren't you fixing the merge conflict here, either??

File: package.json [hunk 4]
```diff
@@ -21,6 +21,10 @@
   "dependencies": {
     "dotenv": "^10.0.0",
     "express": "^4.17.1",
+<<<<<<< HEAD
+    "fs": "0.0.1-security",
```
Comment (ka930) [inline package.json:25]: fs doesn't need to be installed

File: public/createSurvey.html [hunk 5]
```diff
@@ -100,60 +133,32 @@
...
-                                            <option value="2">Number</option>
-                                            <option value="3">Rating</option>
-                                          </select>
-                                    <label class="form-label" style="font-size:30px;color: white;">Question Label</label>
-                                     <input placeholder="Type question label here" style="width: 100%;"></input>
-                                    </div></div>`);
-                                    qLength++;
+            $("#body-card").append("<div class='card-body'><div class='col-md-12'><div class='card' style='background-color:#91BAD6;margin:2%;'><div class='card-body'><select class='form-select' aria-label='Default select example'><option selected>Please Choose What Type of Survey Question</option><option value='1'>Input</option><option value='2'>Rating</option><option value='3'>Number</option></select><label class='form-label;' style='color:white;font-size:30px'>Question</label><input placeholder='Please Type In Question' style='width:100%;'></input></div></div></div></div>");
         });
-        $("#submit").on('click', submitForm);
```
Comment (ka930) [inline public/createSurvey.html:126]: why are you removing the code?

**Pull request closed without merging**